<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnitCellPropertyWidget</class>
 <widget class="QWidget" name="UnitCellPropertyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="3">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxb">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="labelb">
       <property name="text">
        <string>b</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labela">
       <property name="text">
        <string>a</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxa">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="labelc">
       <property name="text">
        <string>c</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxc">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelalpha">
       <property name="text">
        <string>a</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxalpha">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>90.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="labelbeta">
       <property name="text">
        <string>a</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxbeta">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>90.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxgamma">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>90.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="labelgamma">
       <property name="text">
        <string>a</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Spacegroup</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter or select space group and &lt;/p&gt;&lt;p&gt;press Enter to validate your choice&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Chemical Formula </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_ChemicalFormula"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Z </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBox_Z"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>HKL integer tolerance</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="hklTolerance">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.200000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>hklTolerance</zorder>
 </widget>
 <tabstops>
  <tabstop>doubleSpinBoxa</tabstop>
  <tabstop>doubleSpinBoxb</tabstop>
  <tabstop>doubleSpinBoxc</tabstop>
  <tabstop>doubleSpinBoxalpha</tabstop>
  <tabstop>doubleSpinBoxbeta</tabstop>
  <tabstop>doubleSpinBoxgamma</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
